phpunit-integration.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
```

Comment:
most of this file is a direct copy of phpunit.xml . The biggest difference is that in this file we do not have anything about `MYSQL_HOST`, so the code can use the real host as configured in the environment.

At some point we might want to set up two separate hosts, one for manual testing in local and a different one for these automated tests.